packages/openapi-generator/README.md

```diff
@@ -31,47 +31,45 @@ USAGE
-
-  --[no-]strictNaming                                     By default, the generation fails, when there are duplicate
-                                                          names for operations and/or path parameters after transforming
-                                                          them to camel case. Set this to true to enable unique name
-                                                          generation. The names will then be generated by appending
-                                                          numbers.
-
-  --tsConfig=<path/to/tsconfig.json>                      Replace the default `tsconfig.json` by passing a path to a
-                                                          custom config. By default, a `tsconfig.json` is only
-                                                          generated, when transpilation is enabled (`--transpile`).
+  -i, --input=<path/to/input>
+      (required) Specify the path to the directory or file containing the OpenAPI service definition(s) to generate 
+      clients for. Accepts Swagger and OpenAPI definitions as YAML and JSON files. Throws an error if the path does not 
+      exist.
+
+  -o, --outputDir=<path/to/output>
+      (required) Specify the path to the directory to generate the client(s) in. Each client is generated into a 
+      subdirectory within the given output directory. Creates the directory if it does not exist. Customize subdirectory 
+      naming through `--serviceMapping`.
+
+  -t, --transpile
+      Transpile the generated TypeScript code. When enabled a default `tsconfig.json` will be generated and used. It emits 
+      `.js`, `.js.map`, `.d.ts` and `.d.ts.map` files. To configure transpilation set `--tsconfig`.
+
+  --clearOutputDir
+      Remove all files in the output directory before generation. Be cautious when using this option, as it really removes 
+      EVERYTHING in the output directory.
+
+  --include=<glob/to/include>
+      Include files matching the given glob into the root of each generated client directory.
+
+  --packageJson
+      When set to true, a `package.json`, that specifies dependencies and scripts for transpilation and documentation 
+      generation is generated.
+
+  --perServiceConfig=<path/to/per-service-config.json>
+      Set the path to a file containing the configuration per service. The configuration allows to set a `directoryName` 
+      and `packageName` for every service. It also makes sure that names do not change between generator runs. If a 
+      directory is passed, a `per-service-config.json` file is read/created in this directory.
+
+  --[no-]strictNaming
+      By default, the generation fails, when there are duplicate names for operations and/or path parameters after 
+      transforming them to camel case. Set this to true to enable unique name generation. The names are then be generated 
```

Comment:
I thought you have set this to `false` so that we get `enable unique name generation.`